validation/jinja2/validation/_wrong_buttons.html

```diff
@@ -0,0 +1,42 @@
+{#
+      Copyright (C) 2021 Jolene Poulin <jcpoulin@ualberta.ca>
+
+      This program is free software: you can redistribute it and/or modify
+      it under the terms of the GNU Affero General Public License as
+      published by the Free Software Foundation, either version 3 of the
+      License, or (at your option) any later version.
+
+      This program is distributed in the hope that it will be useful,
+      but WITHOUT ANY WARRANTY; without even the implied warranty of
+      MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+      GNU Affero General Public License for more details.
+
+      You should have received a copy of the GNU Affero General Public License
+      along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#}
+
+{% macro button_col(recording, csrf_token, request) %}
+
+    {% if recording.quality == "wrong word" %}
```

Comment:
I feel like duplication here can be simplfied by specifying each button exactly _once_ and toggling the `button--neutral-solid` class in the template.

---

**NOTE**: you do NOT need to implement the following, but I might as well mention it:

 I believe the more approriate markup for this toggle would be to use a `<input type="checkbox">`, wrap it in a `<label>` and use the checkbox's `checked` attribute to toggle the appropriate class on the button. I don't expect this implementation in this PR though!

Unfortunately, [no browser supports the `:has()` pseudo-class][https://caniuse.com/css-has] which would make this CSS possible:

```css
label.wrong-speaker-button:has(> input[type=checkbox][checked]) {
    color: var(--solid-button-text-color);
    background-color: var(--solid-button-background);
}
```